tests/drop_counters/test_drop_counters.py

```diff
@@ -85,13 +86,19 @@ def setup(duthost, testbed):
     for dut_port, neigh in mg_facts['minigraph_neighbors'].items():
         neighbor_sniff_ports.append(mg_facts['minigraph_port_indices'][dut_port])
 
+    for vlan_name in mg_facts["minigraph_vlans"].keys():
+        match = re.findall("\d{1,4}", vlan_name)
+        if match:
+            configured_vlans.append(int(match[0]))
+
     setup_information = {
         "port_channel_members": port_channel_members,
         "vlan_members": vlan_members,
         "rif_members": rif_members,
         "dut_to_ptf_port_map": mg_facts["minigraph_port_indices"],
         "combined_drop_counter": combined_drop_counter,
-        "neighbor_sniff_ports": neighbor_sniff_ports
+        "neighbor_sniff_ports": neighbor_sniff_ports,
+        "vlans": configured_vlans
```

Comment:
to have cleaner diff when adding more elements I suggest to put a comma even if it is a last element